google_guest_agent/oslogin.go

```diff
@@ -198,6 +198,7 @@ func updateSSHConfig(sshConfig string, enable, twofactor, skey bool) string {
 		}
 	}
 	authorizedKeysUser := "AuthorizedKeysCommandUser root"
+	trustedUserCAKeys := "TrustedUserCAKeys /etc/ssh/trustedca.pub"
```

Comment:
What happens if the system doesn't have such a file: `/etc/ssh/trustedca.pub` ?

Reply:
I tested this. It works perfectly fine. Even more, when you add the file, SSHD finds it and starts using with no restart needed

Reply:
As I explained this on a sync meeting, the change is totally independent  to the rest of modifications and is safe to have it merged already.